tidy3d/components/medium.py

```diff
@@ -5180,6 +5181,190 @@ class SurfaceImpedanceFitterParam(Tidy3dBaseModel):
+    """Modified Hammerstad surface roughness model.
+
+    Note
+    ----
+
+        The power loss compared to smooth surface is described by:
+
+        .. math::
+
+            1 + (RF-1) * \\frac{2}{\\pi}\\arctan(1.4\\frac{R_q^2}{\\delta^2})
+
+    Note
+    ----
+    This model is based on:
+
+        Y. Shlepnev, C. Nwachukwu, "Roughness characterization for interconnect analysis",
+        2011 IEEE International Symposium on Electromagnetic Compatibility,
+        (DOI: 10.1109/ISEMC.2011.6038367), 2011.
+
+        V. Dmitriev-Zdorov, B. Simonovich, I. Kochikov, "A Causal Conductor Roughness Model
+        and its Effect on Transmission Line Characteristics", Signal Integrity Journal, 2018.
+    """
+
+    rq: pd.PositiveFloat = pd.Field(
+        ...,
+        title="RMS Peak-to-Valley Height",
+        description="RMS peak-to-valley height (Rq) of the surface roughness.",
+        units=MICROMETER,
+    )
+
+    roughness_factor: float = pd.Field(
+        2.0,
+        title="Roughness Factor",
+        description="Expected maximal increase in conductor losses due to roughness effect. "
+        "Value 2 gives classic Hammerstad equation with maximal possible increase in conductor loss 2.",
+        gt=1.0,
+    )
+
+    def roughness_correction_factor(self, frequency: float, skin_depths: float) -> complex:
+        """Complex-valued roughness correction factor applied to surface impedance.
+
+        Notes
+        -----
+            The roughness correction factor should be causal. It is multiplied to the
+            surface impedance of the lossy metal to account for the effects of surface roughness.
+
+        Parameters
+        ----------
+        frequency : float
+            Frequency to evaluate roughness correction factor at (Hz).
+        skin_depths : float
+            Skin depths of the lossy metal that is frequency-dependent.
+
+        Returns
+        -------
+        complex
+            The causal roughness correction factor evaluated at ``frequency``.
+        """
+        normalized_laplace = -1.4j * (self.rq / skin_depths) ** 2
+        sqrt_normalized_laplace = np.sqrt(normalized_laplace)
+        causal_response = np.log(
+            1 + 2 * sqrt_normalized_laplace / (1 + normalized_laplace)
+        ) + 2 * np.arctan(sqrt_normalized_laplace)
+        return 1 + (self.roughness_factor - 1) / np.pi * causal_response
+
+
+class HuraySurfaceRoughness(AbstractSurfaceRoughness):
+    """Huray surface roughness model.
+
+    Note
+    ----
+
+        The power loss compared to smooth surface is described by:
+
+        .. math::
+
+            \\frac{A_{matte}}{A_{flat}} + \\frac{3}{2}\\sum_i f_i/[1+\\frac{\\delta}{r_i}+\\frac{\\delta^2}{2r_i^2}]
+
+    Note
+    ----
+    This model is based on:
+
+        J. Eric Bracken, "A Causal Huray Model for Surface Roughness", DesignCon, 2012.
+    """
+
+    relative_area: pd.PositiveFloat = pd.Field(
+        1,
```

Comment:
is it bounded by 0 and 1?

Reply:
I read somewhere that "In general, set Amatte/Aflat = 1 , and it is set larger than one only for very high roughness". So it can exceed 1.